packages/website/src/components/types.ts

```diff
@@ -14,14 +14,27 @@ export interface RuleDetails {
 
 export type TabType = 'code' | 'tsconfig' | 'eslintrc';
 
+export type ConfigFileType =
+  | 'ts'
+  | 'tsx'
+  | 'js'
+  | 'jsx'
+  | 'd.ts'
+  | 'cjs'
+  | 'mjs'
+  | 'cts'
+  | 'mts';
```

Comment:
could this be `ts.Extension` instead?

Reply:
my issue with using ts in this context is that we do not have access to it yet, we have to wait for  playground to be fully initialized, and to do so we have to know what extensions has to be choosen,
addtionally we will have to parse remove some options from it `TsBuildInfo` and `Json`